app/controllers/items_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ItemsController < ApplicationController
+  def index
+    @items = Item.all
+  end
+
+  def show
+    @item = Item.find(params[:id])
+  end
+
+  def new
+    @merchant = Merchant.find(params[:merchant_id])
+  end
+
+  def create
```

Comment:
Looks like you've got some code here without a corresponding test. Make sure you are using TDD in your development process!